### ISSUE TITLE
Deprecated bundle

### ISSUE THREAD

**Issue by TucoSolo** [OpenCollar/opencollar#24]
What is your Second Life name?
Tuco Solo

What steps will reproduce the problem?
Use Updater 3.931. Select Custom. There's a button "OldAndBu". Hit it.

What is the expected output?
No message, or actually, not this button.

What do you see instead?
The Updater reports:
"OpenCollarUpdater Dev/Beta/RC - 3.931: The OldAndBusted bundle is deprecated and must be removed."

What version of the collar (or other item) are you using?
OpenCollar Updater 3.931

Please provide any additional information below.
(Not sure whether this is a bug, defect, hidden feature or something that was forgotten. But as a user, I got confused.)

**Comment by WendyStarfall** [comment]
that button resembles the "OldAndBusted" bundle, that bundle has stuff inside that /must/ be removed; all that changed for 3.930 is that instead of an X character the button has now a recycle symbol; if you click it, it tells you in chat what this bundle is all about: "The OldAndBusted bundle is deprecated and must be removed."

**Comment by TucoSolo** [comment]
Yet, as a user, it confuses me.
As a developer: could we make such mandatory bundles hidden in some way?

**Comment by WendyStarfall** [comment]
this question never came up until now (over two years of the bundle updater being in circulation) and I believe it is good to show that the updater has the two required bundles: core and oldandbusted; they show right in the custom menu (which is already just used by people who want /custom/) that there even are those two required bundles. who's interested might wanna dig into the contents and inspect those further.

in a nutshell this helps to make the bundle updater more self-explanatory. to wrap this up: don't worry, nobody else will be confused. this question has a 0.00000325% chance of recurrence within a window of 27 months taking into account the amount of updaters distributed during that time.
